src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -173,6 +173,14 @@ public void setDesiredState(SwerveModuleState desiredState) {
         desiredSpeedLog.append(desiredState.speedMetersPerSecond);
     }
 
+    public void park(boolean reversed) {
+        if(reversed){
+          turningPIDController.setReference(-Math.PI/4, ControlType.kPosition);  
```

Comment:
I think we still want to use calculateSetpoint first (like on line 168) to optimize the directions the the wheels turn.

Reply:
Looked at the code with Josh, calculateSetPoint is for determining the remainder after a full rotation.

He is adding a comment to make that more clear.